autowarefoundation/autoware.universe
# refactor(behavior_velocity_planner): use pluginlib

## Description

Split behavior_velocity_planner scene module into an independent package by using pluginlib. See https://github.com/autowarefoundation/autoware.universe/issues/3724 for details. This PR focuses on splitting the package, so no documentation or parameter files have changed. These will be covered in other PRs.

## Related links

https://github.com/autowarefoundation/autoware.universe/issues/3724
[TIER IV INTERNAL LINK](https://tier4.atlassian.net/browse/RT2-723)

## Tests performed

- Check autonomous driving in simple planning simulator.
- Check scenarios in scenario simulator.

## Notes for reviewers

I split the package.xml file, so please check the maintainer, author, and depend tags.

## Interface changes

None

## Effects on system behavior

None (It should not be affected)

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.


## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.


## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.


After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/
